WordPress/openverse
# Fix sketchfab loading state error

## Fixes



Fixes #3729 by @krysal 

## Description



Fixes the display logic for showing the single image result loading skeleton by introducing a computed `showLoadingState` conditional that checks for the presence of a sketchfab identifier. 

This PR _also_ fixes the display of single sketchfab results. Previously, the viewer was only rendering ~300px wide (you can view an example here: https://openverse.org/image/70a225e6-05e5-498a-862a-cbd0ed44d9c4?q=dog). This is because our logic for displaying images correctly relies on object-fit which only applies to replaced elements like images and videos, not the Sketchfab iframe.

We should probably have an additional visual regression test for single Sketchfab results to prevent this in the future. I'll create a new issue for that.

## Testing Instructions


1. Checkout the PR and `j f` to run the frontend locally
2. Make an image search and filter by sketchfab.
3. Visit a single result; observe it now displays correctly

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
